src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/streaming_endpoint.py

```diff
@@ -64,6 +64,41 @@ def remove_akamai_access_control(client, account_name, resource_group_name, stre
     return client.update(resource_group_name, account_name, streaming_endpoint_name, streaming_endpoint)
 
 
+def update_streaming_endpoint(client, resource_group_name, account_name, streaming_endpoint_name,  # pylint: disable=too-many-locals
```

Comment:
![image](https://user-images.githubusercontent.com/8594276/44421639-89f3e380-a557-11e8-9d5e-bf95b3605619.png)
Investigate this error... This won't be a common scenario... btw it shouldn't throw an error like the one shown above

Reply:
I tried to reproduce it but I couldn't, it seems to be a conectivity error